.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
+    - package-ecosystem: 'github-actions'
```

Comment:
The indentation for YAML seems weird. Possible misconfiguration?

Reply:
I'd suggest overriding some rules for YAML files: it seems like two-space indentation is much more common. We won't get that many changes to those files if we do. I tried this and got no weird indentation:

```json
"overrides": [
	{
		"files": ["*.yml", "*.yaml"],
		"options": {
			"tabWidth": 2
		}
	}
]
```

Reply:
Done!